webcommon/libs.nashorn/build.xml

```diff
@@ -22,4 +22,10 @@
 <project basedir="." default="build" name="webcommon/libs.nashorn">
     <description>Builds, tests, and runs the project org.netbeans.libs.nashorn</description>
     <import file="../../nbbuild/templates/projectized.xml"/>
+    <import file="../../nbbuild/templates/common.xml"/>
+
+    <!-- For testing purposes, build distribution of JS parser in a separate directory -->
```

Comment:
Unzipping it into `test/classes` directory might make it available to others via standard `<test>` dependency.